include/elf/arc-reloc.def

```diff
@@ -497,3 +497,24 @@ ARC_RELOC_HOWTO(ARC_NPS_CMEM16, 78, \
                 replace_bits16, \
                 dont, \
                 ( ME ( S + A )))
+
+ARC_RELOC_HOWTO(ARC_JLI_SECTOFF, 63, \
+                2, \
+                10, \
+                replace_jli, \
+                bitfield, \
+                ( ( S - JLI ) ))
+
+ARC_RELOC_HOWTO(ARC_JLI_SECTOFF_SIMM12, 79, \
+                2, \
+                12, \
+                replace_simm12_20, \
+                bitfield, \
+                ( ( ( S - JLI ) >> 2 ) ))
+
+ARC_RELOC_HOWTO(ARC_JLI_32, 80, \
```

Comment:
I am not really convinced that we did a good job here. Please can u explain once again why do u need this reloc. I find a bit strange a reloc can require two formulas to be solved.